frontend/src/bundles/common/pages/budgets/budget-details/components/doughnut-chart-card/doughnut-chart-card.tsx

```diff
@@ -1,65 +1,80 @@
+import { type IconProp } from '@fortawesome/fontawesome-svg-core';
 import classNames from 'classnames';
 
-import BlueMurseIcon from '~/assets/img/blue-murse-icon.svg';
-import OrangeMurseIcon from '~/assets/img/orange-murse-icon.svg';
-import { DoughnutChart } from '~/bundles/common/components/components';
-import { DoughnutChartCartVariant } from '~/bundles/landing/enums/enums';
+import { DoughnutChart, Icon } from '~/bundles/common/components/components';
+import { dateToShortStringHelper } from '~/bundles/common/helpers/helpers';
+import { type DoughnutChartCartVariant } from '~/bundles/landing/enums/enums';
 
 import styles from './styles.module.scss';
 
 type Properties = {
     title: string;
     date: string;
-    transaction_num: number;
-    transaction_type: string;
-    transaction_sum: string;
+    transaction_num?: number;
```

Comment:
this type should requered i think

Reply:
Fields that do not contain this information are in the object. They are required